radix-transactions/src/builder/manifest_namer.rs

```diff
@@ -1031,24 +1017,6 @@ impl<A: TryInto<DynamicResourceAddress, Error = E>, E: Debug> ResolvableResource
     }
 }
 
-impl<'a> ResolvableResourceAddress for &'a str {
```

Comment:
Why have we removed these??

It means that manifest builder methods such as `take_non_fungibles_from_worktop` which opt into a `ResolvableResourceAddress` can't take a named resource address (via the name) any more?

Reply:
Maybe we should add a test to demonstrate this so someone else isn't tempted to remove these in future :D